### PR TITLE
Fix typos in lib/ssh

### PR DESCRIPTION
**Pull request by kianmeng**

### CODE REVIEW

File: lib/ssh/src/ssh_xfer.erl [hunk 1]
```diff
@@ -409,7 +409,7 @@ encode_open_flags(Flags) ->
       fun (read) -> ?SSH_FXF_READ;
 	  (write) -> ?SSH_FXF_WRITE;
 	  (append) -> ?SSH_FXF_APPEND;
-	  (creat) -> ?SSH_FXF_CREAT;
```
Comment (lisztspace) [inline lib/ssh/src/ssh_xfer.erl:412]: Not a typo, but naming inherited from Ye Olde Standards.

File: lib/ssh/src/ssh_xfer.erl [hunk 2]
```diff
@@ -474,7 +474,7 @@ decode_open_flags(Vsn, F) when Vsn =< 3 ->
 		 {?SSH_FXF_READ, read},
 		 {?SSH_FXF_WRITE, write},
 		 {?SSH_FXF_APPEND, append},
-		 {?SSH_FXF_CREAT, creat},
```
Comment (lisztspace) [inline lib/ssh/src/ssh_xfer.erl:477]: Not a typo, but naming inherited from Ye Olde Standards.

File: lib/ssh/src/ssh_sftpd.erl [hunk 3]
```diff
@@ -623,7 +623,7 @@ sftp_to_erlang_flag(write, Vsn) when Vsn == 3;
 sftp_to_erlang_flag(append, Vsn) when Vsn == 3;
 				      Vsn == 4 ->
     append;
-sftp_to_erlang_flag(creat, Vsn) when Vsn == 3;
```
Comment (lisztspace) [inline lib/ssh/src/ssh_sftpd.erl:626]: Not a typos, but naming inherited from Ye Olde Standards.

File: lib/ssh/src/ssh_sftp.erl [hunk 4]
```diff
@@ -1509,7 +1509,7 @@ open_mode3(Modes) ->
 	     {_, _, true} ->
 		 [append];
 	     {true, false, false} ->
-		 [write, creat, trunc];
```
Comment (lisztspace) [inline lib/ssh/src/ssh_sftp.erl:1512]: Not a typo, but naming inherited from Ye Olde Standards.

**Review by lisztspace — CHANGES_REQUESTED** [review]
Thanks for your excellent corrections to quite a few typos in code and documentation.  Theer are, however, a few that actually are not typos and should remain.  Please see comments.

File: lib/ssh/test/ssh_sftp_SUITE.erl [hunk 5]
```diff
@@ -399,7 +399,7 @@ open_close_file(Config) when is_list(Config) ->
 
     ok = open_close_file(Sftp, SftpFileName, [read]),
     ok = open_close_file(Sftp, SftpFileName, [write]),
-    ok = open_close_file(Sftp, SftpFileName, [write, creat]),
```
Comment (lisztspace) [inline lib/ssh/test/ssh_sftp_SUITE.erl:402]: Not a typo.  Name inherited from old standards.

File: lib/ssh/test/ssh_relay.erl [hunk 6]
```diff
@@ -313,7 +313,7 @@ listen(Parent, LSock) ->
     do_listen(Parent, LSock).
 
 do_listen(Parent, LSock) ->
-    %% So annoying there is no select-like sematic for this
+    %% So annoying there is no select-like semantics for this
```
Comment (lisztspace) [inline lib/ssh/test/ssh_relay.erl:316]: Changed to "semantics" as "semantic" is adjective.

**Review by lisztspace — CHANGES_REQUESTED** [review]

**Review by lisztspace — COMMENTED** [review]
Thanks for the update.  This has now been added for internal testing.

**Comment by lisztspace** [comment]
Thank you for your contribution to increasing readability.

**Comment by kianmeng** [comment]
:tada: :tada: :tada: :tada: :tada:

**Pull request merged**